nemo_skills/inference/model/tool_call.py

```diff
@@ -63,7 +64,7 @@ def __init__(
         self.call_interpreter = ChatCompletionCallInterpreter()
         self.response_formatter = ChatCompletionResponseFormatter()
 
-    async def _execute_tool_call(self, tool_call):
+    async def _execute_tool_call(self, tool_call, request_id: str):
         ## TODO(sanyamk): The correct key format needs to be cohesive with other formatters.
```

Comment:
We can remove this TODO and we should standardize always using the tool calls format, whether it comes from manual parsing on native support from model servers.